velox/docs/functions/presto/tdigest.rst

```diff
@@ -0,0 +1,113 @@
+==================
+T-Digest Functions
+==================
+
+Velox uses the modern KLL digest algorithm for its primary ``approx_percentile``
```

Comment:
Are you suggesting that T-digest are for legacy workloads, but users should prefer modern KLL digest algorithms with approx_percentile ?

Reply:
Yup, The Velox implementation of approx_percentile() uses KLL internally not T-Digest or Q-digest